osx/setup-db.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Sets up the SolarNetwork PostgreSQL DB
+
+WORKSPACE=$1
+
+# Make sure that a workspace has been specified
+if [ -z "$WORKSPACE" ]; then
+  "A SolarNetwork workspace must be specified"
+  exit 1
+fi
+
+# Check that PostgreSQL is installed
+type -P psql &>/dev/null && echo "Configuring psql"  || { echo "$psql command not found."; exit 1; }
+
+# Set up the PostgreSQL database
+dropdb solarnetwork
+dropuser solarnet
+
+dropdb solarnet_unittest
+dropuser solarnet_test
+
+createuser -AD solarnet
+psql -U postgres -d postgres -c "alter user solarnet with password 'solarnet';"
+createdb -E UNICODE -l C -T template0 -O solarnet solarnetwork
+createlang plv8 solarnetwork
+psql -U postgres -d solarnetwork -c "CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;"
+
+createuser -AD solarnet_test
+psql -U postgres -d postgres -c "alter user solarnet_test with password 'solarnet_test';"
+createdb -E UNICODE -l C -T template0 -O solarnet_test solarnet_unittest
+createlang plv8 solarnet_unittest
+psql -U postgres -d solarnet_unittest -c "CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;"
+
+# Setup base database
+cd $WORKSPACE/solarnetwork-central/net.solarnetwork.central.datum/defs/sql/postgres
+
+# for some reason, plv8 often chokes on the inline comments, so strip them out
+sed -e '/^\/\*/d' -e '/^ \*/d' postgres-init-plv8.sql | psql -d solarnetwork -U postgres
+psql -d solarnetwork -U solarnet -f postgres-init.sql
+psql -d solarnetwork -U solarnet -f postgres-init-data.sql
```

Comment:
Looks like you added the loading of `postgres-init-data.sql` but that wasn't part of the original script. The DDL no longer works, and would require updates to get it working. I don't think it is necessary to load.